packages/myst-cli/src/project/fromPath.ts

```diff
@@ -127,8 +127,11 @@ function indexFileFromPages(pages: (LocalProjectFolder | LocalProjectPage)[], pa
   if (!indexFile) indexFile = matcher('.md');
   if (!indexFile) indexFile = matcher('.tex');
   if (!indexFile) indexFile = matcher('.ipynb');
+  if (!indexFile) indexFile = matcher('.json');
   if (!indexFile) [indexFile] = files.filter((file) => extname(file) === '.md');
   if (!indexFile) [indexFile] = files.filter((file) => extname(file) === '.tex');
+  if (!indexFile) [indexFile] = files.filter((file) => extname(file) === '.ipynb');
```

Comment:
Was this a bug?

Reply:
I think so!

Reply:
(FWIW, this was the last extension type, so it would just get picked up in the final `[indexFile] = files;`. Definitely better as updated, since it's more explicit, but I don't think there is any worry that we inadvertently changed someones project with this change.)